src/__tests__/looksrare/executeCollectionOrder.test.ts

```diff
@@ -2,45 +2,70 @@ import { expect } from "chai";
 import { utils } from "ethers";
 import { ethers } from "hardhat";
 import { setUpContracts, SetupMocks, getSigners, Signers } from "../helpers/setup";
+import { ownerOf, balanceOf } from "../helpers/tokens";
 import { LooksRare } from "../../LooksRare";
-import { SupportedChainId, CollectionType } from "../../types";
+import { SupportedChainId, CollectionType, CreateMakerCollectionOfferInput } from "../../types";
 
 describe("execute collection order", () => {
   let mocks: SetupMocks;
+  let lrUser1: LooksRare;
   let signers: Signers;
+  let lrUser2: LooksRare;
+  let collectionOfferInput: CreateMakerCollectionOfferInput;
+
   beforeEach(async () => {
     mocks = await setUpContracts();
     signers = await getSigners();
+    lrUser1 = new LooksRare(SupportedChainId.HARDHAT, ethers.provider, signers.user1, mocks.addresses);
+    lrUser2 = new LooksRare(SupportedChainId.HARDHAT, ethers.provider, signers.user2, mocks.addresses);
 
-    const tx = await mocks.contracts.transferManager
-      .connect(signers.user1)
-      .grantApprovals([mocks.addresses.EXCHANGE_V2]);
-    await tx.wait();
-  });
-  it("execute collection order", async () => {
-    const lrUser1 = new LooksRare(SupportedChainId.HARDHAT, ethers.provider, signers.user1, mocks.addresses);
-    const lrUser2 = new LooksRare(SupportedChainId.HARDHAT, ethers.provider, signers.user2, mocks.addresses);
-
-    let tx = await lrUser2.approveErc20(lrUser2.addresses.WETH);
-    await tx.wait();
-    tx = await lrUser1.approveAllCollectionItems(mocks.contracts.collectionERC721.address);
-    await tx.wait();
-
-    const { maker } = await lrUser2.createMakerCollectionOffer({
+    collectionOfferInput = {
       collection: mocks.contracts.collectionERC721.address,
       collectionType: CollectionType.ERC721,
       subsetNonce: 0,
       orderNonce: 0,
       startTime: Math.floor(Date.now() / 1000),
       endTime: Math.floor(Date.now() / 1000) + 3600,
       price: utils.parseEther("1"),
-    });
+    };
+
+    let tx = await lrUser1.grantTransferManagerApproval().call();
+    await tx.wait();
+
+    tx = await lrUser2.approveErc20(lrUser2.addresses.WETH);
```

Comment:
`lrUser2.addresses.WETH` sounds wrong even though it's technically correct, how about `mocks.addresses.WETH`